### PR TITLE
cody: update readme with more info for user

### PR DESCRIPTION
**Pull request by abeatrix**
1. Add more context to README.md.  The introduction was generated by Claude so feel free to change it as you see fit.

Marketplace intro:

image

2. Also updated the BETA tag to `experimental` as per https://sourcegraph.slack.com/archives/C04NPH6SZMW/p1679500800430569?thread_ts=1679500061.813439&cid=C04NPH6SZMW

image

## Test plan

Docs update

See https://github.com/sourcegraph/sourcegraph/tree/bee/cody-docs/client/cody for result page.

## App preview:

- [Web](https://sg-web-bee-cody-docs.onrender.com/search)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.

### CODE REVIEW

File: client/cody/package.json [hunk 1]
```diff
@@ -1,15 +1,22 @@
 {
   "name": "cody",
   "private": true,
-  "displayName": "Cody",
+  "displayName": "Cody for Enterprise",
   "version": "0.0.6",
-  "publisher": "sourcegraph",
+  "publisher": "Sourcegraph",
```
Comment (novoselrok) [inline client/cody/package.json:6]: Shouldn't this be lowercase? It is in `client/vscode/package.json`.

File: client/cody/README.md [hunk 2]
```diff
@@ -1,18 +1,57 @@
...
+- Cody can instantly answer questions, generate documentation and unit tests on demand, suggest and even write code for you. With Cody, you'll spend less time searching for answers and more time achieving flow state - staying focused, in the zone, and building at peak productivity.
+- Cody enables you to code faster, with more confidence, and avoid vulnerabilities by putting the world of code at your fingertips with Cody - your AI-powered programming companion.
+
+## Highlighted features
+
+- Answer questions about your codebase instantly
+- Generate documentation and unit tests on demand
+- Write code snippets and prototypes for you
+- Translate comments and functions in your code between languages
+- Find and fix vulnerabilities or anti-patterns in your code
```
Comment (novoselrok) [inline client/cody/README.md:35]: I'd remove the last line. We don't want to promise to w.r.t to code security initially.

File: client/cody/README.md [hunk 3]
```diff
@@ -1,18 +1,57 @@
...
+
+Program with an expert by your side - code with confidence using Cody.
```
Comment (novoselrok) [inline client/cody/README.md:37]: I don't think we need this line.

File: client/cody/package.json (hunk 1 above)
Comment (abeatrix) [reply client/cody/package.json]: If we update it to lowercase, the publish name will also be lower case in marketplace it seems, which is different than our vs code extension preview 🤔 
image
image

Comment (novoselrok) [reply client/cody/package.json:6]: Probably, just in case it is case-sensitive.

Comment (abeatrix) [reply client/cody/package.json:6]: Done!

File: client/cody/package.json [hunk 4]
```diff
@@ -1,15 +1,22 @@
...
   "publisher": "sourcegraph",
   "license": "Apache-2.0",
   "icon": "resources/cody.png",
-  "description": "",
+  "description": "Your programming sidekick powered by AI and Sourcegraph's code search and intelligence.",
   "categories": [
     "Other"
   ],
+  "keywords": [
+    "openai",
```
Comment (novoselrok) [inline client/cody/package.json:14]: I'd replace this with just `developer tools` and `ai`. We can add more later.

**Review by novoselrok — APPROVED** [review]

**Pull request merged**